src/components/WalletInit/CreateWallet/MnemonicBackupImportanceModal.stories.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react'
+import {storiesOf} from '@storybook/react-native'
+
+import {withModalProps} from '../../../../storybook/decorators'
+import MnemonicBackupImportanceModal from './MnemonicBackupImportanceModal'
+
+storiesOf('MnemonicBackupImportanceModal', module)
```

Comment:
This story is not showing for me in the navigator.

Reply:
ive havent been committing the file generated by storybook, which adds all the stories
it was causing conflicts, and i didnt want to deal with it anymore
`yarn storybook` will add all the stories and start a web interface